include/crocoddyl/core/activations/smooth-2norm.hpp

```diff
@@ -49,8 +49,12 @@ class ActivationModelSmooth2NormTpl : public ActivationModelAbstractTpl<_Scalar>
    * @param[in] nr   Dimension of the residual vector
    * @param[in] eps  Smoothing factor (default: 1.)
    */
-  explicit ActivationModelSmooth2NormTpl(const std::size_t& nr, const Scalar& eps = Scalar(1.))
-      : Base(nr), eps_(eps){};
+  explicit ActivationModelSmooth2NormTpl(const std::size_t& nr, const Scalar& eps = Scalar(1.)) : Base(nr), eps_(eps) {
+    if (eps <= Scalar(0.)) {
```

Comment:
Same. here. I think this should be `eps < Scalar(0)`

Reply:
In this case the motivation is slightly different.
If we accept `eps=0`, then this function becomes the quadratic one, which it is already defined.

We could accept the zero condition even if this is a weird choice since we could use the other activation model.

Reply:
1) what is defined is the squarenorm, not the norm.
2) But that is beside the point. Let  the user use this function in the manner that is useful to his/her application. We should not restrict from our side. Please see the previous comment

Reply:
done